src/utils/template-utils.ts

```diff
@@ -0,0 +1,29 @@
+import type { AST } from "svelte-eslint-parser"
```

Comment:
I prefer to add it to `src/utils/ast-utils.ts` rather than adding a new file. Could you move the util functions?